src/Data/Vec/Properties.agda

```diff
@@ -381,6 +381,12 @@ cast-sym eq {xs = x ∷ xs} {ys = y ∷ ys} xxs[eq]≡yys =
   let x≡y , xs[eq]≡ys = ∷-injective xxs[eq]≡yys
   in cong₂ _∷_ (sym x≡y) (cast-sym (suc-injective eq) xs[eq]≡ys)
 
+≈-cong′ : ∀ {f-len : ℕ → ℕ} (f : ∀ {n} → Vec A n → Vec B (f-len n))
```

Comment:
Is it `Vec A (f-len n)`?

Reply:
I see what I have missed. It's correct to have `B` in the type. However, I do feel like it is better to place `≈-cong′` in `Data.Vec.Relation.Binary.Equality.Cast.CastReasoning`.

Reply:
yeah, that was my first impulse as well, but it's unfortunately not possible to place it there, precisely because of that `B`. to do it, we should first convert the `A` module parameter into a variable. shall I do it?

Reply:
Yes, I think it's worth it to turn the module parameter `A` into `variable`s in order to group `≈-cong′` and other combinators together.

I think using `variable`s makes `≈-sym` and `≈-trans` uglier since `{A}` will have to be explicitly supplied, but I don't have a better solution.

Reply:
Unfortunately, if we turn the module parameter into a variable in `Data.Vec.Relation.Binary.Equality.Cast.CastReasoning` we will have to make this is `v3.0` PR as that is breaking change. I suggest therefore we stick with the current location?

Reply:
Ah except... that the module parameter is implicit. In which case I think it's fine to move it there :+1:

Reply:
done. I think the signatures were left unchanged (there was no unwanted repositioning of implicit args), please review

Reply:
ha! I didn't realize that the type could be added to `Sym` and `Trans`.